### PR TITLE
feat: paginate audit logs

### PR DESCRIPTION
**Pull request by chetannn**

### CODE REVIEW

File: pages/settings/audits.tsx [hunk 1]
```diff
@@ -1,30 +1,58 @@
...
+  const [page, setPage] = useState(0);
+  const {
+    data: auditLogs,
+    fetchNextPage,
+    isLoading,
+    status,
+    isFetchingNextPage,
+    hasNextPage
+  } = trpc.audits.paginate.useInfiniteQuery(
+    { limit: 20 },
```
Comment (dahal) [inline pages/settings/audits.tsx:26]: Lets do the limit of `10` so it nicely fits on the page.

File: pages/settings/audits.tsx [hunk 2]
```diff
@@ -1,30 +1,58 @@
...
 
       <div className="w-full lg:w-3/5">
-        <AuditLogs logs={logs} user={user} />
+        {isLoading ? (
+          "loading..."
+        ) : (
+            auditLogs?.pages.map((page, index) => (
+              <Fragment key={index}>
+                <AuditLogs logs={page.logs} user={user} />
+              </Fragment>
```
Comment (dahal) [inline pages/settings/audits.tsx:43]: Looks like re-rendering breaks the UI, instead we can use state and `useEffect` hook on `AuditLog` component?
Screen Shot 2023-02-01 at 6 07 29 PM

**Review by dahal — CHANGES_REQUESTED** [review]
Overall great job. Thanks for contributing. 🤗

File: pages/settings/audits.tsx (hunk 1 above)
Comment (chetannn) [reply pages/settings/audits.tsx:26]: sure

File: pages/settings/audits.tsx (hunk 2 above)
Comment (chetannn) [reply pages/settings/audits.tsx:43]: Gonna look on that.

**Pull request closed without merging**